kobuki_ftdi/CMakeLists.txt

```diff
@@ -15,8 +15,8 @@ find_package(ecl_command_line REQUIRED)
 
 # pkg-config packages
 find_package(PkgConfig)
-pkg_search_module(libusb REQUIRED libusb)
-pkg_search_module(libftdi REQUIRED libftdi)
+pkg_check_modules(libusb REQUIRED libusb)
```

Comment:
Reference: https://cmake.org/cmake/help/v3.12/module/FindPkgConfig.html

Not sure why this was needed though. Let's leave it as is so we're not introducing change without knowing why (debugging pain). If the former still works, and there is a reason for this change, it can be PR'd later.

Reply:
Switched back to `pkg_search_module`.  @Karsten1987 , let us know why you needed to make this change.  We can do a follow-up PR if you need this change to build on macOS.